### ISSUE TITLE
Feature Request: add new formatter gorgeous

### ISSUE THREAD

**Issue by hidu** [golang/vscode-go#2503]
**Is your feature request related to a problem? Please describe.**
`gorgeous` is a new powerful formatter.
Project is: https://github.com/fsgo/go_fmt

**Describe the solution you'd like**
Add it to "Format Tool" option.

or "Format Tool" can set any other customized tool, rather than the limited ones.

**Comment by hyangah** [comment]
@hidu Custom formatter support had been available since v0.24.0, but this extension assumes the formatter can work with the input supplied with STDIN, and output to STDOUT. I am not familiar with the `gorgeous` tool. Does it support STDIN based file content? If so, try `"go.formatTool": "gorgeous"` . 

I prepared a change that made this feature more visible. If it's merged in in the current state and your tool supports STDIN/STDOUT, users may enable it like 

```
"go.formatTool": "custom",
"go.alternateTools": { "customFormatter": "gorgeous" }
```

**Comment by hyangah** [comment]
The new setting tweak will be available in the next release. Until then, use `"go.formatTool": "<custom_tool_name>"`.
Closing.
